File: ProgramasJava/src/secao6/Exercicio06.java
package secao6;

import java.util.Scanner;

public class Exercicio06 {

	public static void main(String[] args) {

		int c;
		float n, e = 0, salario, valor_hora = (float) 10.00;

		Scanner teclado = new Scanner(System.in);

		System.out.print("Informe o código: ");
		c = teclado.nextInt();

		System.out.print("Informe as horas trabalhadas: ");
		n = teclado.nextFloat();

		if (n > 50) {
			e = (float) (n - 50) * (float) 20.00;
			salario = (50 * valor_hora) + e;
			System.out.printf("Salário total: R$ %.2f\n", salario);
			System.out.printf("Salário total: R$ %.2f\n", e);

		} else {
			salario = n * valor_hora;
			System.out.printf("Salário total R$ %.2f\n", salario);
			System.out.printf("Salário total R$ %.2f\n", e);
		}

		teclado.close();
	}
}

File: ProgramasPython/ProgramasPython/secao06/Exercicio06.py
e = 0

c = int(input("Código: "))
n = float(input("Horas trabalhadas: "))

if n > 50:
    e = (n - 50) * 20.00
    salario = (50 * 10.00) + e
    print("Salário total: {0:.2f}".format(salario))
    print("Salário excedente: {0:.2f}".format(e))

else:
    salario = (n * 10.00)
    print("Salário total: {0:.2f}".format(salario))
    print("Salário excedente: {0:.2f}".format(e))